src/CitizenProject/CitizenProjectFactory.php

```diff
@@ -75,10 +78,18 @@ public function createFromCitizenProjectCreationCommand(CitizenProjectCreationCo
             $command->problemDescription,
             $command->proposedSolution,
             $command->requiredMeans,
-            $command->getCommittee(),
+            $command->getCommitteeSupports()->toArray(),
             $command->getAddress() ? $this->addressFactory->createFromNullableAddress($command->getAddress()) : null
         );
 
+        $committeeSupports = $citizenProject->getCommitteeSupports()->toArray();
+        foreach ($committeeSupports as &$object) {
+            if ($object instanceof Committee) {
+                $object = new CitizenProjectCommitteeSupport($citizenProject, $object);
+            }
+        }
+        $citizenProject->setCommitteeSupports(new ArrayCollection($committeeSupports));
```

Comment:
This call here looks wrong. The factory should not be responsible for "re-initializing" the project properties.